package.json

```
@@ -10,8 +10,9 @@
   "author": "labforward GmbH",
   "main": "dist/validation.js",
   "scripts": {
-    "build": "del-cli dist && yarn build:schema && tsc -p tsconfig.build.json",
+    "build": "del-cli dist && yarn build:schema && tsc -p tsconfig.build.json && yarn copy-dts",
     "build:schema": "ts-node src/compile.ts",
+    "copy-dts": "copyfiles -u 1 'src/types.d.ts' dist",
```

should we generalize this?
```suggestion
    "copy-dts": "copyfiles -u 1 'src/**/*.d.ts' dist",
```

---

I originally had it like this, but preferred to be explicit because a) it's rather unlikely we will split up types in multiple files, b) using a glob for a single file feels wrong and c) if we ever do extend types into multiple files we are likely to have some added folder folder structure which this command currently does not accommodate for anyway